### PR TITLE
for test - fixes staking v2

### PR DESCRIPTION
**Pull request by AdoAdoAdo**

### CODE REVIEW

**Review by sasurobert — APPROVED** [review]

File: cmd/node/config/config.toml [hunk 1]
```diff
@@ -12,42 +12,42 @@
    StartInEpochEnabled = true
 
    # SCDeployEnableEpoch represents the epoch when the deployment of smart contracts will be enabled
-   SCDeployEnableEpoch = 2
+   SCDeployEnableEpoch = 1
```
Comment (iulianpascalau) [inline cmd/node/config/config.toml:15]: I would revert (value 2) the following flags: `SCDeployEnableEpoch`, `BuiltInFunctionsEnableEpoch`, `RelayedTransactionsEnableEpoch`, `TransactionSignedWithTxHashEnableEpoch`, `MetaProtectionEnableEpoch`,  `AheadOfTimeGasUsageEnableEpoch`, `EpochEnable for the second configuration`

File: cmd/node/config/systemSmartContractsConfig.toml [hunk 2]
```diff
@@ -4,9 +4,9 @@
     MinUnstakeTokensValue = "10000000000000000000" #10eGLD
     UnBondPeriod = 250
     MinStepValue = "100000000000000000000"
-    StakeEnableEpoch = 2
-    StakingV2Epoch = 2
-    DoubleKeyProtectionEnableEpoch = 2
+    StakeEnableEpoch = 1
+    StakingV2Epoch = 1
```
Comment (iulianpascalau) [inline cmd/node/config/systemSmartContractsConfig.toml:8]: I would revert (value 2) the following flags: `StakingV2Epoch`, `DoubleKeyProtectionEnableEpoch`, `DelegationManagerSystemSCConfig.EnabledEpoch`,  `DelegationSystemSCConfig.EnabledEpoch`

Comment (AdoAdoAdo) [reply cmd/node/config/systemSmartContractsConfig.toml]: done

File: cmd/node/config/config.toml (hunk 1 above)
Comment (AdoAdoAdo) [reply cmd/node/config/config.toml]: done

**Review by sasurobert — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**